File: python/min-cost-to-move-chips-to-same-position.py
from typing import List
class Solution:
    def minCostToMoveChips(self, position: List[int]) -> int:
        return min(
            len(list(filter(lambda x: x%2 == 0, position))),
            len(list(filter(lambda x: x%2 == 1, position)))
        )

File: go/min_cost_to_move_chips_to_same_position.go
package leetcode

func minCostToMoveChips(position []int) int {
	even, odd := 0, 0
	for _, pos := range position {
		if pos%2 == 0 {
			even++
		} else {
			odd++
		}
	}
	if even < odd {
		return even
	} else {
		return odd
	}
}
